torch/_inductor/lowering.py

```diff
@@ -3489,22 +3488,25 @@ def get_x_interp(y):
     )
 
 
+@register_lowering(aten.reflection_pad1d_backward)
 @register_lowering(aten.reflection_pad2d_backward)
-def reflection_pad2d_backward(grad_output, x, padding):
-    assert len(padding) == 4
-    left, right, top, bot = padding
+@register_lowering(aten.reflection_pad3d_backward)
+def _reflection_padnd_backward(grad_output, x, padding):
+    dim = len(padding) // 2
 
-    *_, h, w = x.get_size()
-    h = V.graph.sizevars.evaluate_static_shape(h) - 1
-    w = V.graph.sizevars.evaluate_static_shape(w) - 1
+    dhw = x.get_size()[-dim:]
+    dhw = [V.graph.sizevars.evaluate_static_shape(h) - 1 for h in dhw]
```

Comment:
I know it's pre-existing but do we need static shapes here? `index_range_condition` even uses `index_expr` so it seems like expressions are supported.

Reply:
Static shapes are not needed. I checked and it works just fine with symbolic shapes.